packages/modules/pricing/src/migrations/Migration20241212190401.ts

```diff
@@ -0,0 +1,11 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20241212190401 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      `UPDATE price_list_rule SET attribute = 'customer.groups.id' WHERE attribute = 'customer_group_id';`
```

Comment:
q: Why did we end up renaming the attribute? Feel like this is an awkward key to have

Reply:
The attributes on all our rules based engines are based on what the cart shape looks like - eg: `items.product.id`, `items.subtotal`, etc. This change brings it into alignment. Prevents us from having to mutate the pricing context wherever needed and relying on a singular shape. 

Important to note though that this won't be seen in the presentation layer, for the users they'd just see "Customer Groups"